.gitignore

```diff
@@ -20,12 +16,11 @@ lib-cov
 
 # Coverage directory used by tools like istanbul
 coverage
-*.lcov
 
 # nyc test coverage
 .nyc_output
 
-# Grunt intermediate storage (https://gruntjs.com/creating-plugins#storing-task-files)
+# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
```

Comment:
Why https to http?

Reply:
The new ignore file is taken from Gatsby JS example, previous one was from a React Project.  This comment has changed between the two files. It will not have any effect on the expected outcome

> Why https to http?

The new ignore file is taken from Gatsby JS example, previous one was from a React Project.  This comment has changed between the two files. It will not have any effect on the expected outcome